spec/core_functions/color/hsl/one_arg/special_functions/expected_output.css

```diff
@@ -0,0 +1,25 @@
+no-alpha {
+  calc-1: hsl(calc(1), 2%, 3%);
+  calc-2: hsl(1, calc(2%), 3%);
+  calc-3: hsl(1, 2%, calc(3%));
+  var-1: hsl(var(--foo), 2%, 3%);
+  var-2: hsl(1, var(--foo), 3%);
+  var-3: hsl(1, 2%, var(--foo));
+  multi-argument-var-1-of-2: hsl(var(--foo) 50%);
+  multi-argument-var-2-of-2: hsl(0 var(--foo));
+  multi-argument-var-1-of-1: hsl(var(--foo));
+}
+
+alpha {
+  calc-1: hsl(calc(1), 2%, 3%, 0.4);
+  calc-2: hsl(1, calc(2%), 3%, 0.4);
+  calc-3: hsl(1 2% calc(3%)/0.4);
+  calc-4: hsl(1 2% 3%/calc(0.4));
+  var-1: hsl(var(--foo), 2%, 3%, 0.4);
+  var-2: hsl(1, var(--foo), 3%, 0.4);
+  var-3: hsl(1 2% var(--foo)/0.4);
+  var-4: hsl(1 2% 3%/var(0.4));
+  multi-argument-var-1-of-2: hsl(var(--foo) 50%/0.4);
+  multi-argument-var-2-of-2: hsl(0 var(--foo)/0.4);
+  multi-argument-var-1-of-1: hsl(var(--foo)/0.4);
+}
```

Comment:
Yes, it is; see [the proposal](https://github.com/sass/language/blob/master/accepted/color-4-rgb-hsl.md), specifically the sections on special number strings.

We pass calls involving `var()` and `calc()` through unchanged because we can't (in general) resolve them to SassScript values at compile-time, since their values depend on information that's only available at render-time.